.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "4.0"
+      }
+    }]
+  ],
+  "plugins": [
+    "add-module-exports",
+    "transform-es2015-modules-commonjs"
```

Comment:
`transform-es2015-modules-commonjs`: Shouldn't that already be done by `babel-preset-env` + `node 4.0` target?